src/pybind/mgr/mgr_module.py

```diff
@@ -1281,7 +1288,10 @@ def open_db(self) -> Optional[sqlite3.Connection]:
             self.create_mgr_pool()
         uri = f"file:///{self.MGR_POOL_NAME}:{self.module_name}/main.db?vfs=ceph";
         self.log.debug(f"using uri {uri}")
-        db = sqlite3.connect(uri, check_same_thread=False, uri=True)
+        try:
+            db = sqlite3.connect(uri, check_same_thread=False, uri=True, autocommit=True)
```

Comment:
I'm a little puzzled by this line as it sets `autocommit` to true but the overall commit says you want to disable autocommit. I'm guessing this is intentional but it's not clear to me why.

Reply:
Thanks a lot for catching that. On my test machine I've been using python 3.6 which is why I have this try/except to begin with. I screwed up the autocommit variable setting when rewriting commits for the PR. (Don't code when tired :sleeping: )